bears/general/SpaceConsistencyBear.py

```diff
@@ -1,38 +1,56 @@
-from coalib.bearlib import deprecate_settings
 from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
 from coalib.bears.LocalBear import LocalBear
 from coalib.results.Diff import Diff
 from coalib.results.Result import Result
+from coalib.bearlib.aspects.Formatting.Blankline import NewlineAtEOF
+from coalib.bearlib.aspects.Formatting.Spacing import (
+    Indentation,
+    TrailingSpace
+)
 
 
-class SpaceConsistencyBear(LocalBear):
+
+class SpaceConsistencyBear(
+    LocalBear,
+    aspect={
+        'detect': [
+            Indentation, 
+            NewlineatEOF,
+            TrailingSpace
+        ]
+    },
+    languages=['Python']):
+    
+    @map_setting_to_aspect(
+        enforce_newline_at_EOF = NewlineAtEOF,
+        allow_trailing_spaces = TrailingSpace,
+        indent_size = Indentation,
+        use_spaces = Indentation
+    )
     LANGUAGES = {'All'}
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_FIX = {'Formatting'}
-
-    @deprecate_settings(indent_size='tab_width')
     def run(self,
             filename,
             file,
-            use_spaces: bool,
-            allow_trailing_whitespace: bool=False,
-            indent_size: int=SpacingHelper.DEFAULT_TAB_WIDTH,
-            enforce_newline_at_EOF: bool=True):
+            enforce_newline_at_EOF: bool=True,
```

Comment:
You shouldn't change any of the parameter names here. The point of `@map_setting_to_aspect` is to "convert" the value from aspect/taste and insert it to coala's bear setting here. Also no need to add the default value `True`.